ports/esp8266/modules/ntptime.py

```diff
@@ -26,14 +28,14 @@ def time():
     finally:
         s.close()
     val = struct.unpack("!I", msg[40:44])[0]
-    return val - NTP_DELTA
+    return val - __NTP_DELTA
 
 
 # There's currently no timezone support in MicroPython, and the RTC is set in UTC time.
-def settime():
-    t = time()
+def settime(timezone: int = 2, host: str = __HOST):
+    t = __time(host)
     import machine
     import utime
 
     tm = utime.gmtime(t)
-    machine.RTC().datetime((tm[0], tm[1], tm[2], tm[6] + 1, tm[3], tm[4], tm[5], 0))
+    machine.RTC().datetime((tm[0], tm[1], tm[2], tm[6] + 1, tm[3] + timezone, tm[4], tm[5], 0))
```

Comment:
There's currently no timezone support in MicroPython (as per the comment) and I don't think this is the right way to add it.